index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
+    <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
+    <title>My Portfolio</title>
+</head>
+<body>
```

Comment:
- [ ] Your page has a white margin at the top. Remove this margin.
![Screenshot from 2021-09-16 04-34-37](https://user-images.githubusercontent.com/65501535/133535561-e03489bb-f690-433e-9edb-80b7de937e0d.png)

Reply:
https://www.loom.com/share/321f7f7aaa624f31850d11afbe3587d3